src/control/cmd/dmg/pool.go

```diff
@@ -92,7 +92,7 @@ func (trf *tierRatioFlag) UnmarshalFlag(fv string) error {
 	for _, trStr := range strings.Split(fv, ",") {
 		tr, err := strconv.ParseFloat(strings.TrimSpace(strings.Trim(trStr, "%")), 64)
 		if err != nil {
-			return errors.Wrapf(err, "invalid tier ratio %s", trStr)
+			return errors.Errorf("invalid tier ratio %q", trStr)
```

Comment:
Why do you remove the `Wrapf()` which seems OK for me.

Reply:
+1

Reply:
same as above reply.